clients/ctl/admin-ui/src/features/dataset/ClassifiedDataCategoryDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, ButtonProps } from "@fidesui/react";
+import { MultiValue, Select } from "chakra-react-select";
+
+import { DataCategory } from "~/types/api";
+
+import {
+  DataCategoryDropdown,
+  Props as DataCategoryDropdownProps,
+} from "./DataCategoryInput";
+
+// TODO: just making up a structure until we have something real from the API
+interface DataCategoryWithConfidence extends DataCategory {
+  confidence: number;
+}
+
+interface Props extends DataCategoryDropdownProps {
+  mostLikelyCategories: DataCategoryWithConfidence[];
+}
+
+const ClassifiedDataCategoryDropdown = ({
+  mostLikelyCategories,
+  onChecked,
+  checked,
+  ...props
+}: Props) => {
+  const menuButtonProps: ButtonProps = {
+    size: "sm",
+    colorScheme: "complimentary",
+    borderRadius: "6px 0px 0px 6px",
+  };
+
+  const options = mostLikelyCategories
```

Comment:
The design is ambiguous, but I noticed that by having this component only receive the most-likely categories, it's a bit unintuitive what happens on the right-half when using the checkbox tree:

<img src="https://user-images.githubusercontent.com/2236777/192069647-f233dd51-f2e1-4cf3-87a9-fcc7ee7be1a7.gif" height="300" >

"Identifiable" doesn't show up in the Select, but "Non-identifiable" does. Maybe we should show "N/A", "Custom", or "Manual" for those that don't have a percent?

Reply:
hmm yeah, it's a strange component. I'm wondering if it'll help once this dropdown gets put in with the actual form which will list all the selected at the bottom. actually, lemme do that now to see if it "feels" any better